README.md

```diff
@@ -115,7 +115,7 @@ const ui = SwaggerUIBundle({...})
 ui.initOAuth({
     clientId: "your-client-id",
     clientSecret: "your-client-secret-if-required",
-    realm: "your-realms",
+    realm: "your-realm",
```

Comment:
Afaict from all the info I've read on `realm`, it's usually like a container around users, roles, etc. So it seems strange to be targeting multiple realms. It for sure won't work with Keycloak. So this is why I think it is better if the example is singular.